src/test/java/com/epam/rp/ui/pages/DashboardPage.java

```diff
@@ -0,0 +1,60 @@
+package com.epam.rp.ui.pages;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.openqa.selenium.support.ui.WebDriverWait;
+
+public class DashboardPage {
+    private final WebDriver driver;
+    private final WebDriverWait wait;
+
+    public By searchField = By.xpath("//input[@placeholder='Search by name']");
+    public By editButton = By.xpath("//span[text()='Edit']");
+    private final By firstDashboard = By.xpath("(//a[@class='gridCell__grid-cell--EIqeC gridCell__align-left--DFXWN dashboardTable__name--t2a89'])[1]");
+
+    public By currentDashboardTitle = By.xpath("//li[@class='pageBreadcrumbs__page-breadcrumbs-item--JhXKI']/span");
```

Comment:
These locators look pretty unstable and contain a lot of details with looks dynamic (e.g. _**name--t2a89**_ and _**item--JhXKI**_)

At the same time, both elements have unique attributes with can be used to more stable and easy to read/maintain locations:
`(//a[contains(@class, 'gridCell')])[1]`
`//li[contains(@class, 'pageBreadcrumbs')]/span`